File: C#/백준/Bronze/1159. 농구 경기/농구 경기.cs
using System;

namespace q{
    class w{
        static void Main(){
            int a = int.Parse(Console.ReadLine());
            int[] aa = new int[26];
            for(int i =0; i<a; i++){
                string b = Console.ReadLine();
                aa[(int)b[0]-97]++;
            }
            int check=0;
            for(int i=0; i<26; i++){
                if(aa[i]>=5){
                    Console.Write((char)(i+97));
                    check=1;
                }
            }
            if(check==0){
                Console.Write("PREDAJA");
            }
        }
    }
}

File: Java/백준/Bronze/1159. 농구 경기/농구 경기.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt();
        int[] aa = new int[26];
        for(int i =0; i<a; i++){
            String b = in.next();
            aa[(int)b.charAt(0)-97]++;
        }
        int check=0;
        for(int i =0; i<26; i++){
            if(aa[i]>=5){
                System.out.print((char)(i+97));
                check++;
            }
        }
        if(check==0){
            System.out.print("PREDAJA");
        }
    }
}